source/languages/shell/generate.rb

```diff
@@ -33,7 +33,6 @@
 #
     grammar[:$initial_context] = [
             :comment,
-            :boolean,
```

Comment:
I believe keeping the boolean pattern is the best choice since it gives the editor the ability to highlight it for practical reasons. We can add an additional scope since it is technically a built-in function.

Reply:
Ok, I have no objections giving true/false their own color, merely tagging them as constants is technically wrong, likewise the inclusion in logical_expression_context.